### ISSUE TITLE
Published 3.2.6 Podspec doesn't declare tvOS platform

### ISSUE THREAD

**Issue by justindhill** [segmentio/analytics-ios#579]
It looks like when 3.2.6 was published, its podspec.json may not have been generated from the Analytics.podspec at `HEAD` of `dev`. It doesn't include a declaration of support for tvOS.

Link to the podspec.json in the CocoaPods Specs repo:
https://github.com/CocoaPods/Specs/blob/master/Specs/Analytics/3.2.6/Analytics.podspec.json

**Comment by f2prateek** [comment]
Correct, 3.2.6 doesn't support tvOS, but the next release will. Or you can use `master`

**Comment by justindhill** [comment]
Ahh, ok. Thanks for the quick response.

**Comment by f2prateek** [comment]
For reference here's how our release process works https://github.com/segmentio/analytics-ios/blob/master/CONTRIBUTING.md. So in general once a PR is merged into `dev`, it waits until the next release wednesday to be merged in `master` where it stays untouched for 2 weeks before being declared "stable".
